cmd/fyne_settings/data/gen.go

```diff
@@ -57,5 +57,4 @@ func main() {
 	bundleFile("themeLightPreview", "widgets-light.png", f)
 
 	bundleFile("appearanceIcon", "appearance.svg", f)
-	bundleFile("languageIcon", "language.svg", f)
```

Comment:
This will be used in 2.0... I hope. Can we leave it in? :)

Reply:
Sure thing. We can do that :)